common/src/main/java/com/wynntils/screens/skillpointloadouts/SkillPointLoadoutsScreen.java

```diff
@@ -509,6 +450,60 @@ public boolean doMouseClicked(double mouseX, double mouseY, int button) {
         return super.doMouseClicked(mouseX, mouseY, button);
     }
 
+    /*
+    This is rather complicated
```

Comment:
Can you rewrite this to summarize the behavior rather than how you arrived at it?

Reply:
Yes, but I would like to keep the process to get to it somewhere. It will be useful for if/when smooth scrolling is implemented into the new book design. Any suggestions on where to put it instead?

Reply:
I don't know really. Maybe we should have a wiki page for that?

Reply:
But if others are ok with having this in source, I'll accept it.

Reply:
Let's save this in Discord, if there is a need to check this again, but keep a shorter explanation of the math here.